app/views/system_admin/accredited_providers/confirmations/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="govuk-form-group">
+  <label class="govuk-label"><%= @change_accredited_provider_form.full_name %></label>
```

Comment:
This label should probably sit just above the h1 and below the error summary otherwise it'll look awkward